spec/system/find_sync/syncing_providers_spec.rb

```diff
@@ -42,10 +42,10 @@ def when_the_sync_runs
     TeacherTrainingPublicAPI::SyncAllProvidersAndCoursesWorker.perform_async
   end
 
-  def then_it_updates_the_course_subject_codes
+  def then_it_updates_the_course_subjects
     course = Course.find_by(code: 'ABC1')
 
-    expect(course.subject_codes).to eq(%w[08])
```

Comment:
can we not keep this as `subject_codes` as we defined the helper method to map those?

Reply:
We could, however this reflects what the sync actually does and it doesn't require digging into the course code in order to understand why subjects is set and subject_codes is returned.